### PR TITLE
fix for PDF

### PR DESCRIPTION
**Pull request by VSBalanchuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs [hunk 1]
```diff
@@ -10,9 +11,9 @@ public class AnnualReportPdf : PdfDocument
         private readonly AnnualReport _annualReport;
         private const int TextWidth = 510;
         private const int LeftIndent = 60;
-        private const string FontName = "Times New Roman";
+        private const string FontName = "Calibri";
         private const int BaseFontSize = 10;
-
+
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:16]: Remove white space

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs [hunk 2]
```diff
@@ -189,25 +191,34 @@ protected void DrawText(XGraphics gfx, PdfPage page)
...
                 currentRowY, format);
             currentRowY += 15;
-            gfx.DrawString($"{_annualReport.ListProperty}", font, XBrushes.Black, column1_X, currentRowY, format);
 
-            currentRowY += 20;
+            additionalHeight = _annualReport.ImprovementNeeds == null
+                ? 0
+                : (_annualReport.ImprovementNeeds.Split('\n').Length - 1) * 10;
+            var textArea = new XRect(column1_X, currentRowY, textAreaWidth, 10 + additionalHeight);
+            tf.DrawString((_annualReport.ListProperty != null ? $"{_annualReport.ListProperty}" : "Інформація відсутня"), font, XBrushes.Black, textArea, XStringFormats.TopLeft);
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:209]: We need this white space?

Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:209]: (Line 210)

**Review by Bodasyuk — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs (hunk 1 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs]: Changed

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs (hunk 2 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:208]: Changed

**Review by christykobyletska — APPROVED** [review]

**Review by YuriiDzioba — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs [hunk 3]
```diff
@@ -189,25 +190,33 @@ protected void DrawText(XGraphics gfx, PdfPage page)
...
             gfx.DrawLine(pen, lineStart, currentRowY, lineEnd, currentRowY);
             //////////////////////////////////////////////////////////
             currentRowY += 15;
             gfx.DrawString("Вкажіть перелік майна, що є в станиці:", new XFont(FontName, BaseFontSize, XFontStyle.Italic, options), XBrushes.Black, column1_X, currentRowY, format);
             currentRowY += 20;
-            gfx.DrawString($"{_annualReport.ImprovementNeeds}", font, XBrushes.Black, column1_X, currentRowY, format);
-
+            additionalHeight = _annualReport.ImprovementNeeds == null
+                ? 0
+                : (_annualReport.ImprovementNeeds.Split('\n').Length - 1) * 10;
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:217]: Remove magic numbers

**Review by Bodasyuk — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs [hunk 4]
```diff
@@ -189,25 +190,33 @@ protected void DrawText(XGraphics gfx, PdfPage page)
             page = document.AddPage();
             // Also missing:
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:191]: What is gfx?

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs [hunk 5]
```diff
@@ -10,9 +11,8 @@ public class AnnualReportPdf : PdfDocument
         private readonly AnnualReport _annualReport;
         private const int TextWidth = 510;
         private const int LeftIndent = 60;
-        private const string FontName = "Times New Roman";
-        private const int BaseFontSize = 10;
-
+        private const string FontName = "Calibri";
+        private const int BaseFontSize = 10;
```
Comment (Bodasyuk) [inline EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:15]: Add enter pls

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs (hunk 4 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs:191]: image
This naming is used in official documentation

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs (hunk 5 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs]: Changed

**Review by sharix4 — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs (hunk 3 above)
Comment (VSBalanchuk) [reply EPlast/EPlast.BLL/Services/PDF/Documents/AnnualReportPdf.cs]: changed

**Review by Bodasyuk — APPROVED** [review]

**Pull request merged**